tests/test_export_rms/test_export_rms_volumetrics.py

```diff
@@ -465,6 +465,23 @@ def test_validate_table_against_pydantic_model_before_export(
         exportvolumetrics._validate_table()
 
 
+@inside_rms
+def test_rms_volumetrics_export_config_invalid(
+    mock_project_variable,
+    exportvolumetrics,
+):
+    """Test that an exception is raised if the config is invalid."""
+
+    from fmu.dataio.export.rms.inplace_volumes import export_inplace_volumes
+
+    with mock.patch(  # noqa
```

Comment:
Nesting these with statements fails in python3.8 with an attribute error for `__enter__`. Ruff wants to have them combined, hence the # noqa here